### ISSUE TITLE
[Bug]: create view prompt go error

### ISSUE THREAD

**Issue by dengn** [matrixorigin/matrixone#4505]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 0.5.0
- Hardware parameters: Macbook m1 arm 4c16g
- OS type:
- Others:
```

### Actual Behavior

mysql> create table test(a int, b float, c varchar(255));
Query OK, 0 rows affected (0.03 sec)

mysql> create view hhh as select a,b from test where a=1;
ERROR 1105 (HY000): Internal error: panic runtime error: invalid memory address or nil pointer dereference: goroutine 20 [running]:
runtime/debug.Stack()
	/Users/runner/hostedtoolcache/go/1.18.4/x64/src/runtime/debug/stack.go:24 +0x65
github.com/matrixorigin/matrixone/pkg/common/moerr.NewPanicError({0x4d6a940?, 0x59a1c60})
	/Users/runner/work/matrixone/matrixone/pkg/common/moerr/error.go:110 +0x53
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).ExecRequest.func1()
	/Users/runner/work/matrixone/matrixone/pkg

**Comment by ouyuanning** [comment]
fixed in 0.6

**Comment by aressu1985** [comment]
mysql> create table test(a int, b float, c varchar(255));
Query OK, 0 rows affected (0.35 sec)

mysql> 
mysql> 
mysql>  create view hhh as select a,b from test where a=1;
Query OK, 0 rows affected (0.26 sec)

mysql> 
mysql>
